_resources/port1.0/group/python-1.0.tcl

```diff
@@ -221,7 +228,9 @@ proc python_set_default_version {option action args} {
 options python.branch python.prefix python.bin python.lib python.libdir \
         python.include python.pkgd
 # for pythonXY, python.branch is X.Y
-default python.branch   {[string range ${python.version} 0 end-1].[string index ${python.version} end]}
+default python.branch   {[python_get_defaults branch]}
+default python.language_version {[python_get_defaults language_version]}
+default python.group    {[python_get_defaults group]}
```

Comment:
Some comments are needed explaining what these values mean and what they should be used for.